src/sentry/static/sentry/app/components/groupTombstones.jsx

```diff
@@ -1,61 +1,72 @@
 import React from 'react';
 
 import AlertActions from '../actions/alertActions';
-
+import Avatar from '../components/avatar';
+import EventOrGroupTitle from '../components/eventOrGroupTitle';
 import LoadingError from '../components/loadingError';
 import LinkWithConfirmation from '../components/linkWithConfirmation';
+import TooltipMixin from '../mixins/tooltip';
 
 import ApiMixin from '../mixins/apiMixin';
 
-import {t, tct} from '../locale';
+import {t} from '../locale';
 
 const GroupTombstoneRow = React.createClass({
   propTypes: {
     data: React.PropTypes.object.isRequired,
-    undiscard: React.PropTypes.func.isRequired
+    undiscard: React.PropTypes.func.isRequired,
+    orgId: React.PropTypes.string.isRequired,
+    projectId: React.PropTypes.string.isRequired
   },
+
+  mixins: [
+    TooltipMixin({
+      selector: '.tip'
+    })
+  ],
+
   render() {
-    let {data, undiscard} = this.props;
+    let {data, undiscard} = this.props, actor = data.actor;
+
     return (
       <li className={`group row level-${data.level} type-${data.type}`}>
-        <div className="col-md-7 col-xs-8 event-details">
-          <div>
+        <div className="col-md-10 event-details issue">
+          <div className="event-issue-header">
```

Comment:
since there isn't a group to link to it seemed kind of like it would introduce more special casing / general messiness into that component, but if you think that's better than duplication, i can change it